code/datums/supplypacks.dm

```diff
@@ -1100,6 +1100,16 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	group = "Engineering"
 	access = access_ce
 
+/datum/supply_packs/inflatable
+	name = "Inflatable barriers"
+	contains = list(/obj/item/weapon/storage/box/inflatable,
+					/obj/item/weapon/storage/box/inflatable,
```

Comment:
Yes, I'm sure.

If you use tabs it incorrectly displays depending on the tab size of what you're viewing with, GitHub is an example because it uses 8 instead of (common for most editors) 4.